src/utils/functions/fetchBackendUTXOS.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import { UtxoObject } from '../../types/utxo-object';
+
+/**
+ * Fetch UTXO's for an address or XPUB from backend Blockbook provider
+ * @param backendURL Required. Fully qualified URL for blockbook
+ * @param addressOrXpub Required. An address or XPUB to fetch UTXO's for
+ * @param options Optional. Optional queries based on https://github.com/syscoin/blockbook/blob/master/docs/api.md#get-utxo
+ * @returns JSON object in response, UTXO object array in JSON
+ */
+export async function fetchBackendUTXOS(
```

Comment:
missing return type

Reply:
I'm still figuring out what the return type for this is. The safest way to find it out is to call the API directly.

I created a card for this https://github.com/osiastedian/syscoinjs-lib/projects/1#card-83613546 so that I can proceed and get back to this later